### DISCUSSION TITLE
如何编写样式？

### DISCUSSION THREAD

**Discussion by NriotHrreion** [the1812/Bilibili-Evolved#3529] [提问 / Q&A] [answered]
我打算自己写一个样式，但是在添加的时候一直报错 `Error: 无效的样式代码`

找了很久都没找到样式编写的文档...

**Comment by the1812** [comment, marked as answer]
https://github.com/the1812/Bilibili-Evolved/blob/c23ce91c4c53c84f4e634558834c055b340d4513/src/plugins/style.ts#L13-L23

弄一个 js 文件, 执行的返回值符合这个接口就可以, 例如
```js
({
  name: 'xxx',
  displayName: '显示名称',
  style: `a { color: red; }`,
})
```
